### PR TITLE
docs: Create intro.html

### PR DESCRIPTION
**Pull request by burnyst**
This is a letter of self-introduction.

### CODE REVIEW

File: intro.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <title>자기소개</title>
+</head>
+<body>
+    <h1>자기소개</h1>
+    <div>안녕하세요. 이성훈입니다.</div>
+    <div>서울 관악구 살고 취업 준비를 하고 있어요.</div>
```
Comment (kimdaehee90) [inline intro.html:10]: well done!!

Comment (kimdaehee90) [inline intro.html:10]: just add one more line!

**Review by kimdaehee90 — CHANGES_REQUESTED** [review]
one more please!!

File: intro.html (hunk 1 above)
Comment (burnyst) [reply intro.html:10]: Thanks.

Comment (burnyst) [reply intro.html:10]: ok

Comment (kimdaehee90) [reply intro.html:10]: well done!

**Pull request merged**